lib/core/view.coffee

```diff
@@ -119,6 +121,7 @@ module.exports = class KDView extends KDObject
     @setLazyLoader lazyLoadThreshold  if lazyLoadThreshold
     @setTooltip tooltip               if tooltip
     @setDraggable draggable           if draggable
+    @setAttribute 'draggable', yes    if droppable
```

Comment:
`droppable` means the `element` accepts drops, it should not mean that it is draggable. let's remove this from the framework since all it does is putting the `draggable` attribute.